test/tests/meshgenerators/generated_mesh_generator/tests

```diff
@@ -114,4 +114,34 @@
     design = 'meshgenerators/GeneratedMeshGenerator.md'
     recover = false
   []
+
+  [error_check]
+    issues = '#28998'
+    design = 'meshgenerators/GeneratedMeshGenerator.md'
+    requirement = 'The system shall error when unphysical input combinations are provided to the generated mesh generator, including when'
+    [xmax]
+      type = 'RunException'
+      input = 'generated_mesh_generator.i'
+      cli_args = '--mesh-only Mesh/gmg/xmin=2'
```

Comment:
```suggestion
      cli_args = 'Mesh/gmg/xmin=2 --mesh-only'
```
same for the others. mesh-only should be followed by the mesh name, though it gets it here I think you get a warning